template/v2/Dockerfile

```diff
@@ -17,13 +17,27 @@ ARG NB_GID=100
 
 # https://www.openssl.org/source/
 ARG FIPS_VALIDATED_SSL=3.0.8
+ARG MIN_REQUIRED_MICROMAMBA_VERSION=1.5.11
```

Comment:
Can we update the logics to always upgrade to latest 1.5.x micromamba package instead of 1.5.11? This way we can make sure to always pick up the latest 1.5 patch version in future image releases to fetch bug fixes and CVE fixes

Reply:
sounds good, created a new revision on this